assets/training/model_management/src/run_model_download.py

```diff
@@ -47,7 +48,12 @@ def validate_if_model_exists(model_id):
             model_id = re.sub(NEGATIVE_MODEL_NAME_PATTERN, "-", model_id)
             logger.info(f"Updated model_name = {model_id}")
 
-        models = ml_client_registry.models.get(name=model_id, label="latest")
+        try:
+            models = ml_client_registry.models.get(name=model_id, label="latest")
+        except Exception as e:
+            logger.warning(f"Model with name - {model_id} is not available. Error: {e}")
```

Comment:
Could we also attempt obtaining by version & `max` to get the latest one?

Reply:
Ack